config/data_centers/data_centers.json

```diff
@@ -679,6 +679,36 @@
     "source": "https://docs.aws.amazon.com/global-infrastructure/latest/regions/aws-regions.html",
     "operationalSince": "2021"
   },
+  "hlrs-de-stuttgart": {
+    "provider": "hlrs",
+    "displayName": "HLRS Stuttgart",
+    "zoneKey": "DE-BW",
```

Comment:
This is not a valid zone key, it needs to be the zone key electricity maps uses for the location of the datacenter.